File: core/src/main/java/cn/dozyx/core/utli/SampleUtil.java
package cn.dozyx.core.utli;

import java.util.ArrayList;
import java.util.List;

/**
 * @author dozyx
 * @date 2019-10-30
 */
public final class SampleUtil {

    public static List<String> getStrings() {
        return getStrings(50);
    }

    public static List<String> getStrings(int count) {
        return getStrings(count, 0);
    }

    public static List<String> getStrings(int count, int offset) {
        List<String> datas = new ArrayList<>();
        for (int i = 0; i < count; i++) {
            datas.add("sample: " + (i + offset));
        }
        return datas;
    }
}

File: core/src/main/java/cn/dozyx/core/utli/SampleUtil.kt
package cn.dozyx.core.utli

import java.util.ArrayList

/**
 * @author dozyx
 * @date 2019-10-30
 */
object SampleUtil {

    val strings: MutableList<String>
        get() = getStrings(50)

    @JvmOverloads
    fun getStrings(count: Int, tag: String = ""): MutableList<String> {
        val datas = MutableList<String>(count) {
            "sample $tag: $it "
        }
        return datas
    }
}
